styles/blocks/company-life/life.css

```diff
@@ -0,0 +1,26 @@
+.life {
+    display: grid;
+
+    padding: 54px 2.5%;
+
+    background-color: #ffffff;
+
+    grid-column-gap: 5%;
+    grid-row-gap: 40px;
+    justify-content: space-between;
+}
+
+.life__grid-1r {
```

Comment:
While using grid here is kind of justified, I want to emphasize that the classes should named based on semantics. So you need to come up with more meaningful names here.